pkg/test/common_test.go

```diff
@@ -932,7 +436,7 @@ func TestInjectUploadImageWithBasicAuth(t *testing.T) {
 			}
 		}
 
-		img := test.Image{
```

Comment:
TestInjectUploadImageWithBasicAuth should also be under the upload tests in the new folder, right?